prettier.config.js

```diff
@@ -13,4 +13,5 @@ module.exports = {
   insertPragma: false,
   proseWrap: 'preserve',
   htmlWhitespaceSensitivity: 'ignore',
+  endOfLine: 'crlf',
```

Comment:
I'm personally on Ubuntu while I develop and this really shouldn't be specified. I'm guessing you're on Windows? If so, have you double checked that your Git config `git config core.autocrlf` has an appropriate value.

But yeah let's revert that change please I don't want this to be OS specific, this should be figured out upstream